tools/setup_permissions.sh

```diff
@@ -46,6 +46,15 @@ function grant_task_execution() {
          -H "Authorization: token=${ACS_TOKEN}" \
          "${DCOS_URL}/acs/api/v1/acls/dcos:mesos:master:task:user:${LINUX_USER}/users/${SERVICE_ACCOUNT_NAME}/create"
 
+    # XXX 1.10 curerrently requires this mesos:agent permission as well as
+    # mesos:task permission.  unclear if this will be ongoing requirement.
+    # See DCOS-15682
+    curl -k -L -X PUT \
+         -H "Authorization: token=${ACS_TOKEN}" \
+         "${DCOS_URL}/acs/api/v1/acls/dcos:mesos:agent:task:user:${LINUX_USER}" \
```

Comment:
We use ${SERVICE_ACCOUNT_NAME} here, not ${framework_name}-principal.  I'm not excited about shifting to ${framework_name}-principal because it's more work, and using a non-default name has found several bugs.

I've gotten some tests to pretty much work with the acl diffs here.  Can you explain what your additional acl change represents and why we need it, or reference something that does?